nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -86,8 +86,21 @@ controller_server:
     min_x_velocity_threshold: 0.001
     min_y_velocity_threshold: 0.5
     min_theta_velocity_threshold: 0.001
+    progress_checker_plugin: "progress_checker"
```

Comment:
There are 2 other YAMLs as well that need updated in this directory

Reply:
Was not sure if we need to add them even if we can not changing the default. Added anyway.